### PR TITLE
Update wi-fi-settings-android-enterprise.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note per recommendation on #https://github.com/MicrosoftDocs/memdocs/issues/3597

Fixes#https://github.com/MicrosoftDocs/memdocs/issues/3597

### CODE REVIEW

File: memdocs/intune/configuration/wi-fi-settings-android-enterprise.md [hunk 1]
```diff
@@ -197,6 +197,10 @@ Select this option if you're deploying to an Android Enterprise dedicated, corpo
   - **Automatic**: Use a file to configure the proxy server. Enter the **Proxy server URL** that contains the configuration file. For example, enter `http://proxy.contoso.com`, `10.0.0.11`, or `http://proxy.contoso.com/proxy.pac`.
 
     For more information on PAC files, see [Proxy Auto-Configuration (PAC) file](https://developer.mozilla.org/docs/Web/HTTP/Proxy_servers_and_tunneling/Proxy_Auto-Configuration_(PAC)_file) (opens a non-Microsoft site).
+    
+    > [!NOTE]
+    > When a device is marked corporate, the user gets to have little control and all control will be with the policies enforced.
+    > So when  Wi-Fi Profile is pushed for the device, by design Wi-Fi will be made to turn on.
```
Comment (JohanFreelancer9) [inline memdocs/intune/configuration/wi-fi-settings-android-enterprise.md:203]:
```suggestion
    > When a device is marked **Corporate**, the user has little control, and all control will depend on the policies enforced. So when a Wi-Fi Profile is pushed for the device, Wi-Fi will be turned on by design.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

File: memdocs/intune/configuration/wi-fi-settings-android-enterprise.md (hunk 1 above)
Comment (MandiOhlinger) [reply memdocs/intune/configuration/wi-fi-settings-android-enterprise.md:203]: This is my suggested change:

> When a device is marked as corporate during enrollment (organization-owned), policies control device features and settings. Users can be prevented from managing features and settings in the policy.
>
> When a Wi-Fi policy is assigned to devices, then Wi-Fi is enabled, and users can be prevented from turning off Wi-Fi.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@MandiOhlinger Please review and sign off this PR. Thanks

**Review by MandiOhlinger — APPROVED** [review]

**Comment by MandiOhlinger** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**